rice-namenode/CMakeLists.txt

```diff
@@ -1 +1,8 @@
+include_directories(${CMAKE_SOURCE_DIR}/rpcserver)
+
 add_executable(namenode main.cc)
+target_link_libraries( namenode rpcserver )
```

Comment:
Just for the sake of following a consistent style, we should get rid of the spaces inside these parens (lines 4-8)